### PR TITLE
Rover: Unhide two parameters 

### PR DESCRIPTION
**Pull request by khancyr**
unhide default NAVL1_PERIOD by making a define for it ! @gmorph should default be 6 as in param file or 8 as in the function ? 
put CH_7_PWM_TRIGGER on top of its file and make i static const

### CODE REVIEW

File: APMrover2/control_modes.cpp [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #include "Rover.h"
 
+static const int16_t CH_7_PWM_TRIGGER = 1800;
+
 void Rover::read_control_switch()
```
Comment (gmorph) [inline APMrover2/control_modes.cpp]: This uses up 2 more bytes of RAM that we are running out of - not so much in Rover I know but . . .
Why change this?

Comment (khancyr) [inline APMrover2/control_modes.cpp]: @gmorph static const got plenty advantage again define, I will let you google the difference, you will fine better answer than what I could write. I am surprised that it require a little more ram

File: Tools/autotest/default_params/rover.parm [hunk 2]
```diff
@@ -21,7 +21,7 @@ MODE5        2
 MODE6        0
 STEER2SRV_P     1.8
 SIM_GPS_DELAY 1
-NAVL1_PERIOD 6
+NAVL1_PERIOD 8
```
Comment (magicrub) [inline Tools/autotest/default_params/rover.parm]: why is this changing?

Comment (khancyr) [inline Tools/autotest/default_params/rover.parm]: that is the question I ask to @gmorph . in parameter the value is set to 6 but by default the value is set at 8 ... So what it the better ? I don't feel a difference on SITL and gazebo ... so I don't know

Comment (gmorph) [inline Tools/autotest/default_params/rover.parm]: 8 is a better default for rovers.  6 is a little too sharp.

**Pull request merged**